src/data/nat/parity.lean

```diff
@@ -71,7 +71,7 @@ begin
               one_ne_zero, and_self] },
 end
 
-lemma odd_gt_zero (h : odd n) : 0 < n :=
+lemma zero_lt_of_odd (h : odd n) : 0 < n :=
```

Comment:
Or even
```suggestion
lemma odd.pos (h : odd n) : 0 < n :=
```

Reply:
But it's in the `nat` namespace, so that doesn't really help.

Reply:
Maybe it's best to have both `zero_lt_of_odd` and `odd.pos` as synonyms?

Reply:
`nat.odd_pos`? And also have it with `(h : n % 2 = 1)`.